flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/codegen/calls/ScalarOperators.scala

```diff
@@ -79,6 +79,72 @@ object ScalarOperators {
     }
   }
 
+  def generateIn(
+      nullCheck: Boolean,
+      left: GeneratedExpression,
+      right: scala.collection.mutable.Buffer[GeneratedExpression],
+      addReusableCodeCallback: (String, String) => Any)
+    : GeneratedExpression = {
+    val resultTerm = newName("result")
+    val isNull = newName("isNull")
+
+    val topNumericalType: Option[TypeInformation[_]] = {
```

Comment:
@twalthr I think it is not necessary because function will has too specific input params